src/main/java/Duke/Tasks/Disk.java

```diff
@@ -0,0 +1,75 @@
+package Duke.Tasks;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+public class Disk {
+
+    private final String path;
+    private final DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
Perhaps using SCREAMING_SNAKE_CASE for constant names would better match the Java coding standards?